src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const getSum = (a) => {
+    counter++;
+
+    return function(b) {
```

Comment:
You can add 0 as a default value. So if no argument is passed, it would still work.

Reply:
Thank you for your comments. Could you please check again if I did the changes as you requested?